src/qwed_new/api/main.py

```
@@ -444,168 +543,144 @@ async def verify_math(
+                    developer_fields={"is_valid": False, "result": False, "status": "BLOCKED", "warning": "ambiguous", "simplified": str(simplified), "note": "Interpreted using standard order of operations", "original": str(parsed)},
+                )
+            else:
+                simplified = simplify(parsed)
+                try:
+                    value = float(simplified)
+                    dr = DiagnosticResult.verified(
+                        "Expression evaluated",
+                        developer_fields={"is_valid": True, "value": value, "simplified": str(simplified), "original": str(parsed)},
+                        evidence={"value": value, "simplified": str(simplified)},
+                    )
+                except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)